spot-client/src/common/remote-control/xmpp-connection.js

```diff
@@ -304,6 +304,15 @@ export default class XmppConnection {
             });
     }
 
+    /**
+     * Returns whetheer or not the internal XMPP connection is active.
+     *
+     * @returns {boolean}
+     */
+    isConnected() {
+        return this.xmppConnection?.xmpp.connection.connected;
```

Comment:
Can you use a flag or something rather than reaching into the LJM internals?

Reply:
I decided to create a new flag to make the timing of when it gets set to true/false to be more accurate.